#!/usr/bin/env python
PACKAGE = "pose_fuse"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("alpha_", double_t, 0, "low pass filter alpha value",    0.5, 0,   1)
gen.add("outlier_thres_", double_t, 0, "offset param",    0.3, 0.01,   3)
gen.add("offset_start_range", double_t, 0, " offset_start_range",    0.8, 0.3, 3.0)
gen.add("offset_end_range", double_t, 0, " offset_end_range",    0.5, 0.01,   3)
gen.add("offset_reset_max_range", double_t, 0, " offset_reset_max_range",    2.5, 2.0,   3.0)


gen.add("min_range", double_t, 0, " cast to minimum value for range sensor ",    0.17, 0.00,   0.25)
gen.add("range_buffer_size_", int_t, 0, "window size of raw range sensor data",    5, 2,   100)

exit(gen.generate(PACKAGE, "pose_fuse", "dyn_params"))

